### DISCUSSION TITLE
CoApp not recognized in Arc Browser

### DISCUSSION THREAD

**Discussion by orbithammer** [aclap-dev/video-downloadhelper#1584] [Help and Q&A]
### Your question

CoApp not recognized in Arc Browser even though it work with Firefox. Uninstalled and reinstalled with latest version.

1. Arc Browser
2. Win 10
3. 64bits
4. https://github.com/aclap-dev/vdhcoapp/releases/latest/download/vdhcoapp-windows-x86_64-installer.exe
5. `{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Brock",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}`

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"error":"Failed to start native messaging host.","found":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "used_history_button": true,
    "concurrent_downloads_max": 12,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 3
}

**Comment by Mattmjs** [comment]
It's a chromium browser based off chrome, the officially supported browsers are Firefox, Chrome and Edge.
However it does work on some unofficial browsers, for example I have it working in Ungoogled Chromium.
It might be the native messaging json file is not in the right directory for the companion app to see the browser.
In that case you should contact the people that make the Arc browser to see if they can help.

**Reply by orbithammer** [reply]
I see. Thanks for the quick reply.
